src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -174,7 +174,7 @@ export default {
 	}
 
 	//list of navigation items
-	& > ul,
```

Comment:
@pulsejet Are you sure we can remove this rule? Maybe we should keep it, just in case? :thinking: